openff/toolkit/utils/builtin_wrapper.py

```diff
@@ -123,14 +122,12 @@ def assign_partial_charges(
                 strict_n_conformers=strict_n_conformers,
             )
 
-        partial_charges = unit.Quantity(
-            np.zeros((molecule.n_particles)), unit.elementary_charge
-        )
         if partial_charge_method == "zeroes":
-            pass
+            partial_charges = unit.Quantity(
+                molecule.n_atoms * [0], unit.elementary_charge
+            )
```

Comment:
Nice - This lets us drop numpy as a dependency from the whole file!

```suggestion
            partial_charges = unit.Quantity(
                molecule.n_atoms * [0.], unit.elementary_charge
            )
```